app/services/check_answers/applicant_section.rb

```diff
@@ -18,6 +23,11 @@ class ApplicantSection
     attribute :dependants, :boolean
     attribute :child_dependants, :integer
     attribute :adult_dependants, :integer
+    attribute :partner_employed, :boolean
```

Comment:
I'm not sure why we're adding these attributes here? None of these fields belong in the applicant section of check answers, so I don't think they should belong in CheckAnswers::ApplicantSection.